node/src/external/jxcore/DownloadJxcoreLib.ps1

```diff
@@ -0,0 +1,22 @@
+$saveProgressPreference = $ProgressPreference
+$ProgressPreference = "SilentlyContinue"
+
+$jxcoreLibUri = "https://github.com/jasongin/jxcore/releases/download/v0.3.1.2/libjxcore.zip"
```

Comment:
It's generally bad practice to check in large binaries to git repos, which is why I set up this downloader script.

I think the right approach here is for opent2t to fork jxcore and create releases, exactly like I did there. Then we can update that URI to point to opent2t/jxcore instead of jasongin/jxcore.

---
In reply to: [71265486](https://github.com/openT2T/client/pull/1#discussion_r71265486) [](ancestors = 71265486)